.npmignore

```diff
@@ -0,0 +1,2 @@
+# .npmignore file
```

Comment:
This comment is totally unhelpful because the filename `.npmignore` already introduces itself very well.